collector/benchlib/src/measure/perf_counter/unix.rs

```diff
@@ -12,33 +12,82 @@ struct Counters {
     cache_references: Counter,
 }
 
-/// Benchmarks a single function.
-pub fn benchmark_function<F: FnOnce() -> R, R>(
+#[repr(C, align(64))]
+#[derive(Clone)]
+struct CacheAligned(u8);
+
+/// Try to flush the caches to avoid keeping data from previous benchmark iterations in the cache.
```

Comment:
Hmm, I don't like that this is being added with zero evidence that it helps. Do we really need it? If we do, there should be a big comment here explaining that it's untested and should be measured properly at some point.